src/components/CountyMap/CountyMap.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import {
+  ComposableMap,
+  Geographies,
+  Geography,
+  ZoomableGroup,
+} from 'react-simple-maps';
+import { scaleQuantile } from 'd3-scale';
+import { csv } from 'd3-fetch';
+import ReactTooltip from 'react-tooltip';
+import STATE_CENTERS from '../../enums/us_state_centers';
+
+const CountyMap = () => {
+  const [data, setData] = useState([]);
+  const { id: location } = useParams();
+  const state = STATE_CENTERS[location];
+  const counties = require(`./data/${location}.json`);
+
+  useEffect(() => {
+    // https://www.bls.gov/lau/
+    csv('/countyConfigData/sampleData.csv').then(counties => {
+      setData(counties);
+    });
+
+    csv('/countyConfigData/sampleData.csv').then(counties => {
+      setData(counties);
+    });
+  }, []);
+
+  const colorScale = scaleQuantile()
+    .domain(data.map(d => d.unemployment_rate))
```

Comment:
unemployment rate?

Reply:
This is placeholder data for the cloreopath map of counties.